src/bitmessagecli.py

```diff
@@ -160,30 +170,30 @@ def apiData():
     global keysName
     global keysPath
     global usrPrompt
-    
-    BMConfigParser().read(keysPath) #First try to load the config file (the keys.dat file) from the program directory
+
+    BMConfigParser().read(keysPath)  # First try to load the config file (the keys.dat file) from the program directory
 
     try:
-        BMConfigParser().get('bitmessagesettings','port')
+        BMConfigParser().get('bitmessagesettings', 'port')
         appDataFolder = ''
-    except:
-        #Could not load the keys.dat file in the program directory. Perhaps it is in the appdata directory.
+    except BaseException:
```

Comment:
This is not better than bare `except:` IMHO

Reply:
This was autopep8's work. I agree it's not much better. Codacy seems to be happy with BaseException but not Exception, which flags as a bare except. Advice I've seen and follow is to use exception classes based on Exception (or more usually my own exception base class).

Regarding all the bare excepts, I intend to get a good test rig set up and try to predict and detect the exceptions that are actually raised. Without that though I'd just be guessing and will break something.